src/mixins/Themeable.ts

```diff
@@ -175,7 +179,31 @@ export function ThemeableMixin<T extends Constructor<WidgetBase<ThemeablePropert
 				this.recalculateThemeClasses();
 			}
 
-			const appliedClasses = classNames
+			const themeable = this;
+			function classesGetter(this: ClassesFunctionChain) {
```

Comment:
I don't think so, it's got a different scope and also needs to reference the themeable scope.